CVC5 fail to install using this Homebrew tap

This is the output I get (my setup is described below):

```
❯ brew install cvc5
==> Downloading https://formulae.brew.sh/api/formula.jws.json
##################################################################################################################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
##################################################################################################################################################################### 100.0%
==> Fetching cvc5/cvc5/cvc5
==> Downloading https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz
##################################################################################################################################################################### 100.0%
==> Downloading https://files.pythonhosted.org/packages/37/fe/65c989f70bd630b589adfbbcd6ed238af22319e90f059946c26b4835e44b/pyparsing-3.1.1.tar.gz
##################################################################################################################################################################### 100.0%
==> Cloning https://github.com/cvc5/cvc5.git
Cloning into '/Users/anton_1/Library/Caches/Homebrew/cvc5--git'...
==> Checking out tag cvc5-1.0.8
HEAD is now at c8e12cd12 Bump version to 1.0.8
==> Installing cvc5 from cvc5/cvc5
==> python -m venv --system-site-packages --without-pip /usr/local/Cellar/cvc5/1.0.8/libexec
==> python -m pip --python=/usr/local/Cellar/cvc5/1.0.8/libexec/bin/python install /private/tmp/cvc5--tomli-20231126-31341-1iq399g/tomli-2.0.1
==> python -m pip --python=/usr/local/Cellar/cvc5/1.0.8/libexec/bin/python install /private/tmp/cvc5--pyparsing-20231126-31341-1mv6529/pyparsing-3.1.1
==> bash -c export CMAKE_PREFIX_PATH=/usr/local/Cellar/cvc5/1.0.8/libexec:$CMAKE_PREFIX_PATH; ./configure.sh --auto-download --static --prefix\=/usr/local/Cellar/cvc5/1.0.8
Last 15 lines from /Users/anton_1/Library/Logs/Homebrew/cvc5/04.bash:
-- Looking for setitimer - found
-- Performing Test STRERROR_R_CHAR_P
-- Performing Test STRERROR_R_CHAR_P - Failed
CMake Error at cmake/Helpers.cmake:250 (message):
  Could not find module pyparsing for Python version 3.12.  Make sure to
  install pyparsing for this Python version via

  `/usr/local/bin/python3.12 -m pip install pyparsing'.

  Note: You need to have pip installed for this Python version.
Call Stack (most recent call first):
  src/rewriter/CMakeLists.txt:17 (check_python_module)


-- Configuring incomplete, errors occurred!

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/cvc5/homebrew-cvc5/issues
```

My setup is based on macOS 14.1.1 (Sonoma) and here is my Python3 version:
```
❯ brew info python3
==> python@3.11: stable 3.11.6 (bottled)
```

I'll gladly provide any more details that would help debug this issue. Thank you

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -42,5 +42,5 @@ jobs:
         if: always() && github.event_name == 'pull_request'
         uses: actions/upload-artifact@main
         with:
-          name: bottles
+          name: bottle-${{ matrix.os }}
           path: '*.bottle.*'

--- a/Formula/cvc5.rb
+++ b/Formula/cvc5.rb
@@ -3,16 +3,15 @@ class Cvc5 < Formula
 
   desc "Efficient open-source automatic theorem prover for SMT problems"
   homepage "https://cvc5.github.io/"
-  url "https://github.com/cvc5/cvc5.git", tag: "cvc5-1.0.8"
+  url "https://github.com/cvc5/cvc5.git", tag: "cvc5-1.1.1"
   head "https://github.com/cvc5/cvc5.git", branch: "main"
 
   option "with-java-bindings", "Build Java bindings based on new C++ API"
-  option "with-python-bindings", "Build python bindings based on new C++ API"
 
-  depends_on "cmake"
+  depends_on "cmake" => :build
   depends_on "gmp"
   depends_on :java if build.with? "java-bindings"
-  depends_on "python@3.10"
+  depends_on "python@3" => :build
 
   resource "tomli" do
     url "https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz"
@@ -25,7 +24,7 @@ class Cvc5 < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec)
+    venv = virtualenv_create(buildpath/"venv", "python3")
     venv.pip_install resources
 
     command_line = [
@@ -33,10 +32,9 @@ class Cvc5 < Formula
       "--auto-download",
       "--static",
       "--prefix=#{prefix}",
-      "-DCMAKE_FIND_FRAMEWORK=NEVER",
+      "-DPython_EXECUTABLE=#{buildpath}/venv/bin/python",
     ]
 
-    command_line << "--python-bindings" if build.with? "python-bindings"
     command_line << "--java-bindings" if build.with? "java-bindings"
 
     command = Shellwords.join(command_line)

--- a/Formula/cvc5.rb
+++ b/Formula/cvc5.rb
@@ -9,9 +9,9 @@ class Cvc5 < Formula
   option "with-java-bindings", "Build Java bindings based on new C++ API"
 
   depends_on "cmake" => :build
+  depends_on "python@3" => :build
   depends_on "gmp"
   depends_on :java if build.with? "java-bindings"
-  depends_on "python@3" => :build
 
   resource "tomli" do
     url "https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz"

